file/fileRouter.js

```diff
@@ -0,0 +1,63 @@
+const fs = require("fs");
+const path = "./allFiles/"
+/*
+Save a file locally by writing it to a local location
+It's path is to a "allFiles" folder, with a subfolder of the clubName
+    and fileName
+Returns the path of the file
+All inputs are strings
+file must be the contents of the file, which can be accessed by calling 
+    retrieveFiles
+*/
+function postFile(file_id, file) {
```

Comment:
I assume that should be the request object, or at least some kind of streamable object?